tests/testthat/test-borders.R

```diff
@@ -1,11 +1,10 @@
 context("check borders rendering")
 
 skip_on_cran()
-skip_on_ci()
+skip_on_ci() # ?
```

Comment:
why do we need to skip on ci?

Reply:
because it requires MS office to be tested. These scripts are not perfect, they were supposed to run only on my computer (as a very convenient way to be sure the doc are valid and are visually as expected). As suggested, we can get rid of the tests that involve visual testing for now.